docs/vtex-io/Store-Framework-Apps/advanced-components/vtex-store-video.md

```diff
@@ -44,14 +44,14 @@ The Video app allows you to display video assets on your store pages.
 | `type`           | `string`             | Video type.                                                                                                                                                                                            | `undefined`       |
-| `poster`         | `string`             | Cover image URL to be displayed before the video playback.                                                                                                                                             | `undefined`       |
-| `controlsType`   | `enum`               | The type of controls. It can be `custom-vtex`( only works if the video URL represents a HTML5 player ), `native` or `none`.                                                                            | `undefined`       |
-| `autoPlay`       | `boolean`            | Whether the video will start automatically after loaded(`true`) or not(`false`). Note that if the value is `true`, the muted property will automatically be setted to `true`.                          | `false`           |
-| `muted`          | `boolean`            | Whether the video will start with the audio on(`false`) or not(`true`).                                                                                                                                | `false`           |
-| `loop`           | `boolean`            | Whether the video will run in a loop(`true`) or not(`false`).                                                                                                                                          | `false`           |
-| `playsInline`    | `boolean`            | Whether the video will play inline(`true`) or not(`false`).                                                                                                                                            | `false`           |
-| `width`          | `number` or `string` | The width of the video exhibition area. It could be as %(`string`) or pixels(`number`).                                                                                                                | `undefined`       |
-| `height`         | `number` or `string` | The height of the video exhibition area. It could be as %(`string`) or pixels(`number`).                                                                                                               | `undefined`       |
+| `poster`         | `string`             | Cover image URL to be displayed before the video playback. This is only available on the HTML5 player.                                                                                                 | `undefined`       |
+| `controlsType`   | `enum`               | The type of controls. It can be `custom-vtex` (only works if the video URL represents a HTML5 player), `native` or `none`.                                                                             | `undefined`       |
```

Comment:
```suggestion
| `controlsType`   | `enum`               | Video control options. It can be `custom-vtex` (only works if the video URL represents an HTML5 player), `native` or `none`.                                                                             | `undefined`       |
```